torch/ao/utils/__init__.py

```diff
@@ -0,0 +1,109 @@
+from torch import nn
+
+def _module_to_fqn(model, layer, prefix=''):
+    for name, child in model.named_children():
+        new_name = prefix + '.' + name
+        if child is layer:
+            return new_name
+        child_path = _module_to_fqn(child, layer, prefix=new_name)
+        if child_path is not None:
+            return child_path
+    return None
+
+
+def _fqn_to_module(model, path):
+    path = path.split('.')
+    for name in path:
+        model = getattr(model, name, None)
+        if model is None:
+            return None
+    return model
+
+
+def convert(model: nn.Module, mapping: dict = None, config: dict = None) -> nn.Module:
```

Comment:
For eager mode, the API should only take the config for sparsity, the qconfig is an attribute of the module.  The config is identical to the config arg that is passed to the prepare function of the sparsifier.

```
config = {
        ...         'seq.0.linear1': {
        ...             'zero_block_shape': (1, 4)
        ...         'seq.0.linear2: {
        ...             'zero_block_shape': (1, 4)
        ...         }
        ... }
```